wire-library/wire-reflector/src/jvmTest/proto/echo3/echo3.proto

```diff
@@ -0,0 +1,70 @@
+// Copied from https://github.com/juliaogris/reflect/blob/v0.0.7/protos/echo3/echo3.proto
```

Comment:
should we put a copyright header on these?

Reply:
done.